### PR TITLE
Release checklist

### PR DESCRIPTION
**Pull request by awjh-ibm**
Introduce release checklist file

### CODE REVIEW

File: .github/RELEASE_CHECKLIST.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Release Checklist
+
+Before releasing ensure that you have:
+- Run `.release/changelog.sh <PREVIOUS_RELEASE_TAG> <RELEASE_TAG>` to update CHANGELOG.md to contain all commits since the previous tag. Running without the previous release tag will get all commits.
+- Commit the updated changelog via PR with commit message "Preparing for release <RELEASE_TAG>"
+
+Releasing:
+- Go to: https://github.com/hyperledger/fabric-contract-api-go/releases
+- Select "Draft a new release"
+- Tag the version as vX.X.X. Note to include the character v. The repo uses a 3 number semantic versioning system.
```
Comment (heatherlp) [inline .github/RELEASE_CHECKLIST.md]: vX.X.X. sounds a little different to "RELEASE_TAG" - do we need to clarify what is meant? Also you've got an extra full stop after the last X

File: .github/RELEASE_CHECKLIST.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- Give the release a title of "Release <RELEASE_TAG>"
+- Writeup release notes:
+    - Include section called "Release Notes" containing the high-level view of changes made in the version that are of note.
+    - (Optional) Include a section called "Migration Notes" detailing "gotchas" for the user of migrating to the new version.
+    - (Optional) Include a section called "Bug Fixes" detailing important bug fixes in the version. Should be listed as bullet points with a link to the JIRA for that bug.
```
Comment (heatherlp) [inline .github/RELEASE_CHECKLIST.md]: are we planning on having a release notes .md file?

File: .github/RELEASE_CHECKLIST.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+Before releasing ensure that you have:
+- Run `.release/changelog.sh <PREVIOUS_RELEASE_TAG> <RELEASE_TAG>` to update CHANGELOG.md to contain all commits since the previous tag. Running without the previous release tag will get all commits.
+- Commit the updated changelog via PR with commit message "Preparing for release <RELEASE_TAG>"
+
+Releasing:
+- Go to: https://github.com/hyperledger/fabric-contract-api-go/releases
+- Select "Draft a new release"
+- Tag the version as vX.X.X. Note to include the character v. The repo uses a 3 number semantic versioning system.
+- Give the release a title of "Release <RELEASE_TAG>"
+- Writeup release notes:
```
Comment (heatherlp) [inline .github/RELEASE_CHECKLIST.md]: I think writeup is two words

File: .github/RELEASE_CHECKLIST.md (hunk 2 above)
Comment (awjh-ibm) [reply .github/RELEASE_CHECKLIST.md]: I was planning on just using the secton of the github release process like node does

File: .github/RELEASE_CHECKLIST.md (hunk 1 above)
Comment (awjh-ibm) [reply .github/RELEASE_CHECKLIST.md]: This is me trying to tell them how to format the release tag I think I'll move the explanation of release tag structure to the top and make this use release tag

**Pull request merged**